{% extends 'Info/base.html' %}
{% block title %}Help{% endblock %}
{% block content %}

    <br>
     <div style="background:cornsilk" class=" div1 container-fluid">
       <div class=" div row">
        <ul>
            <li><h4 style="line-height: 1.3;"><b>Multivariate Analysis of Protein Polymorphism (MAPP)</b><br>
               Calculates the difference in physicochemical properties between the wild and mutant
               amino acids (AA). Larger the deviation, higher is the chance that the mutation causes
                functional disruption of the protein.<br><b>Prediction:</b> Deleterious/Neutral<br><b>PMID:</b> 15965030</h4></li>
            <li><h4 style="line-height: 1.3;"><b>Protein Analysis through Evolutionary Relationships (PANTHER)</b><br>
                An evolutionary based tool which anticipates the function of hypothetical genes based
                on their evolutionary relationship to experimentally characterized genes with known
                function.<br><b>Prediction:</b> Pathogenic/Deleterious<br><b>PMID:</b> 20015972</h4></li>
            <li><h4 style="line-height: 1.3;"><b>Polymorphism Phenotyping (PolyPhen-1)</b><br>
                Engages a set of empirical rules to determine the functional disruption in presence of
                mutation.<br><b>Prediction:</b> Deleterious/Neutral<br><b>PMID:</b> 12202775</h4></li>

            <li><h4 style="line-height: 1.3;"><b>PolyPhen-2</b><br>
                PolyPhen-2 is different from PolyPhen-1 in terms of features used for prediction and
                uses eight sequence based and three structure-based AA properties for predicting the
                functional and structural damage due to AA substitutions.<br><b>Prediction:</b> Deleterious/Neutral <br><b>PMID:</b> 20354512</h4></li>
            <li><h4 style="line-height: 1.3;"><b>Sorting Intolerant From Tolerant (SIFT)</b><br>
                An evolutionary based algorithm which measures the impact of AA change on the
                function of protein based on physical properties and sequence homology.<br><b>Prediction:</b> Deleterious/Neutral<br><b>PMID:</b> 12824425</h4></li>

            <li><h4 style="line-height: 1.3;"><b>PhD-SNP</b><br>
                PhD-SNP is a support vector machine classifier-based method that predicts
                deleterious SNPs using sequence and profile information.: neural network method based on sequence information.<br><b>Prediction:</b> Pathogenic/Benign<br><b>PMID:</b> 16895930</h4></li>

            <li><h4 style="line-height: 1.3;"><b>Synonymous Non-synonymous Analysis Program (SNAP)</b><br>
                Based on a set of codon-aligned nucleotide sequences which determine synonymous
                and non-synonymous substitution rates.<br><b>Prediction:</b> Non-neutral<br><b>PMID:</b> 17526529</h4></li>

            <li><h4 style="line-height: 1.3;"><b>Site Directed Mutator (SDM)</b><br>
                Calculates a stability score based on environment-specific AA substitution
                frequencies within homologous protein families.<br><b>
                    Prediction:</b> Reduced/Increased stability<br><b>
                    PMID:</b> 21593128, 28525590</h4></li>

            <li><h4 style="line-height: 1.3;"><b>STRUM</b><br>
                A machine learning approach that integrates graph-based signatures, pharmacophore
                properties and experimental conditions and uses this information to calculate changes
                in the stability of protein structure.<br>
                <b>Prediction:</b> Destabilizing/Stabilizing<br>
                <b>PMID:</b> 24281696</h4></li>

            <li><h4 style="line-height: 1.3;"><b>Mutation Cut-off Scanning Matrix (mCSM)</b><br>
                A machine learning approach that integrates graph-based signatures, pharmacophore
                properties and experimental conditions and uses this information to calculate changes
                in the stability of protein structure.<br>
                <b>Prediction:</b> Destabilizing/Stabilizing<br>
                <b>PMID:</b> 24281696</h4></li>

            <li><h4 style="line-height: 1.3;"><b>mCSM-lig</b><br>
                Uses graph-based signatures of wild type structural environment to assess the
                structural ramifications on drug binding due to mutations in the protein.<br>
                <b>Prediction:</b> Destabilizing/Stabilizing<br>
                <b>PMID:</b> 247384129</h4></li>

        </ul>
       </div>
      </div>
      <br><br>
{% endblock %}
